compiler/noirc_evaluator/src/ssa/ssa_gen/mod.rs

```diff
@@ -480,11 +483,11 @@ impl FunctionContext<'_> {
     /// Prepare a slice access.
     /// Check that the index being used to access a slice element
     /// is less than the dynamic slice length.
-    fn codegen_slice_access_check(&mut self, index: ValueId, length: Option<ValueId>) {
+    fn codegen_access_check(&mut self, index: ValueId, length: Option<ValueId>) {
```

Comment:
```suggestion
    fn codegen_access_check(&mut self, index: ValueId, length: ValueId) {
```
nit: This should not need to accept an option anymore

Reply:
Done